erpnext/buying/doctype/purchase_order/purchase_order.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, *args, **kwargs):
 			'target_parent_dt': 'Material Request',
 			'target_parent_field': 'per_ordered',
 			'target_ref_field': 'qty',
-			'source_field': 'stock_qty',
+			'source_field': 'qty',
```

Comment:
Actually I think, `ordered_qty` should be maintained in stock_uom. Because purchase_uom can be changed in PO from MR.